plugin/src/test/java/org/opensearch/ml/rest/RestMLGuardrailsIT.java

```diff
@@ -366,7 +366,11 @@ protected Response registerRemoteModel(String modelGroupName, String name, Strin
             + "  \"description\": \"test model\",\n"
             + "  \"connector_id\": \""
             + connectorId
-            + "\"\n"
+            + "\",\n"
+            + "  \"interface\": {\n"
```

Comment:
Is that a new field in the connector index?

Reply:
See above, it's in model index